os-octavia.te

```diff
@@ -86,6 +87,10 @@ allow haproxy_t sysfs_t:filesystem { mount unmount };
 allow haproxy_t user_tmp_t:dir mounton;
 allow haproxy_t NetworkManager_t:file { open read };
 allow haproxy_t sysfs_t:dir mounton;
+gen_tunable(os_octavia_enable_nsfs, false)
+tunable_policy(`os_octavia_enable_nsfs', `
```

Comment:
I would suggest renaming to "os_haproxy_enable_nsfs" as this is the actual change being enabled so far (like for the dac_override boolean above).

Reply:
done, updated to os_haproxy_enable_nsfs